app/upgrades/v2.0.0/upgrades.go

```diff
@@ -117,6 +129,24 @@ func CreateUpgradeHandler(
 	}
 }
 
+func setAuctionParams(ctx sdk.Context, feeBurnerKeeper *feeburnerkeeper.Keeper, auctionKeeper auctionkeeper.Keeper) error {
+	treasury := feeBurnerKeeper.GetParams(ctx).TreasuryAddress
+	_, data, err := bech32.DecodeAndConvert(treasury)
+	if err != nil {
+		return err
+	}
+
+	auctionParams := auctiontypes.Params{
+		MaxBundleSize:          2,
```

Comment:
they really only want two txs per bundle? we should follow up with them on this as well as all other parameters.